Rakefile

```diff
@@ -6,18 +6,23 @@ Rake.application.load_imports
 
 task('metrics:mutant').clear
 namespace :metrics do
-  task :mutant => :coverage do
-    success = Kernel.system(*%w[
+  task mutant: :coverage do
+    jobs = 4 if ENV.key?('CIRCLE_CI')
+
+    arguments = %w[
       bundle exec mutant
-      --zombie
-      --use rspec
+      --ignore-subject Mutant::Meta*
       --include lib
-      --require mutant
       --since HEAD~1
-      --ignore-subject Mutant::Meta*
-      --
-      Mutant*
-    ]) or fail 'Mutant task is not successful'
+      --require mutant
+      --use rspec
+      --zombie
+    ]
+    arguments.concat(%W[--jobs 4]) if ENV.key?('CIRCLE_CI')
```

Comment:
Have you considered having a default of `0` jobs indicate the number of cores should be chosen? This is the vonention used by parallel. If you adopted those conventions, then maybe you could have `--jobs #{ENV.key?('CIRCLE_CI') ? 4 : 0}`